README.md

```diff
@@ -1,2 +1,36 @@
-# Musical-instrumentals-website
+## Musical-instrumentals-website
 Jump on the Beat
+
+Desktop and Mobile version of Project Musical-instrumentals-website:
+complete with project Home Page and About Page.
+
+
+
+## Built With
+
+- HTML
+- CSS
+- JavaScript
+- Lighthouse
+- Webhint
+- Stylelint
+
+## Getting Started
+To get a local copy up and running follow these simple example steps.
+
+- go to the directory that you want to clone the repository in.
+- right click and choose Git Bash here.
+- type "git clone git@github.com:collinsmezie/Musical-instrumentals-website.git"
+
+### Prerequisites
+- VsCode
+
+### Site published at 
+- Coming soon...
+
+## Authors
+👤 **collins chime **
+
+- GitHub: [@collinsmezie](https://github.com/collinsmezie)
+- Twitter: [@collinsmezie1] collins mezie
+- LinkedIn: [collins mezie]
```

Comment:
- [ ] Please make sure that the README contains an "Acknowledgements" section or similar that gives appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/)